### PR TITLE
[full ci] Revert revert of vicadmin auth change

### PR DESCRIPTION
**Pull request by gigawhitlocks**
Same as #2691 except with a commit that fixes the log collection issue we saw in CI after I merged that change. Check the final commit (4177fa2) for changes.

### CODE REVIEW

**Comment by gigawhitlocks** [comment]
Same as #2691 except with a commit that fixes the log collection issue we saw in CI after I merged that change. Check the final commit (4177fa2) for changes.

**Comment by hmahmood** [comment]
lgtm

**Comment by andrewtchin** [comment]
lgtm

**Comment by gigawhitlocks** [comment]
♻️

File: tests/test-cases/Group9-VIC-Admin/9-2-VICAdmin-CertAuth.robot [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Curl
+    [Arguments]  ${path}
+    ${output}=  Run  curl -sk --cert %{DOCKER_CERT_PATH}/cert.pem --key %{DOCKER_CERT_PATH}/key.pem ${vic-admin}${path}
+    Should Not Be Equal As Strings  ''  ${output}
+    [Return]  ${output}
+
+*** Test Cases ***
+Display HTML    
+     ${output}=  Wait Until Keyword Succeeds  10x  10s  Curl  ${EMPTY}
+     Should contain  ${output}  <title>VCH Admin</title>
```
Comment (anchal-agrawal) [inline tests/test-cases/Group9-VIC-Admin/9-2-VICAdmin-CertAuth.robot]: Not sure if it makes a difference, but -`Contain`.

**Comment by gigawhitlocks** [comment]
Last build only had 1 failure ( https://ci.vmware.run/vmware/vic/5985 ) due to a known issue ( #2769 ) I believe -- can I merge this? @hickeng

**Comment by gigawhitlocks** [comment]
♻️

**Comment by gigawhitlocks** [comment]
♻️

**Comment by gigawhitlocks** [comment]
Last build after successful build was killed by me

**Pull request merged**